### PR TITLE
DM-22531: AuxTel on-sky testing Dec/2019

### PR DESCRIPTION
**Pull request by tribeiro**
Adds, ATAOS, ATDome and ATHexapod configuration for the system as of Dec/2019.

### CODE REVIEW

File: ATAOS/v1/hex_m1_201912.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+correction_frequency: 0.2
+
+m1:
```
Comment (patrickingraham) [inline ATAOS/v1/hex_m1_201912.yaml:3]: Please add a comment saying what these numbers are (and units if there are any).

**Review by patrickingraham — COMMENTED** [review]
Permission to move once comment is addressed.

File: ATAOS/v1/hex_m1_201912.yaml (hunk 1 above)
Comment (tribeiro) [reply ATAOS/v1/hex_m1_201912.yaml:3]: Do we want comments on the configuration? You can find that information on the schema file. https://github.com/lsst-ts/ts_ataos/blob/develop/schema/ATAOS.yaml

Comment (patrickingraham) [reply ATAOS/v1/hex_m1_201912.yaml:3]: Ok you're right. It shouldn't be in two places.

**Review by patrickingraham — APPROVED** [review]

**Pull request merged**